functions/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:20-slim
+
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci --only=production
+
+# Copy only the necessary application files
+COPY webapp.js ./
+COPY index.js ./
+COPY android.js ./
+COPY ios.js ./
+COPY legacy.js ./
+COPY handlers.js ./
+COPY fastify-logger.js ./
```

Comment:
This is now creating 7 layers, combine these to  a single one.
https://docs.docker.com/reference/dockerfile/#source-1
```
COPY \ 
	source1 \
	source2 \
	dest/
```